soql-standalone-parser/src/main/scala/com/socrata/soql/ast/Expression.scala

```diff
@@ -46,8 +46,7 @@ object Expression {
         case FunctionCall(SpecialFunctions.WindowFunctionOver, args) =>
           Seq(args.head).flatMap(findIdentsAndLiterals) ++
```

Comment:
Actually, this whole thing should be rewritten to `val mapped = args.flatMap(findIdentsAndLiterals)
mapped.head :: "over" :: mapped.tail`

Reply:
wait, sorry, this is wrong. it should be:
`findIdentsAndLiterals(args.head) ++ "over" ++ args.tail.flatMap(findIdentsAndLiterals)`